doc/cordova.txt

```diff
@@ -23,6 +23,12 @@ Project Commands
     serve .............................. Run project with a local webserver
                                             (including prepare)
 
+Telemetry Commands
+    telemetry on ....................... Turn telemetry collection on
```

Comment:
Also, we need to document these commands and options here: https://github.com/apache/cordova-cli/blob/master/doc/readme.md